charts/merlin/values.yaml

```diff
@@ -19,6 +19,7 @@ deployment:
       cpu: "1"
       memory: 1Gi
 environment: dev
+deploymentLabelPrefix: "gojek.com/"
```

Comment:
nit: Can the value be made generic? maybe `domain.com/`

Reply:
We use gojek.com because the default value in merlin app is the same: https://github.com/gojek/merlin/blob/main/api/config/config.go#L46